components/brave_rewards/browser/rewards_p3a.cc

```diff
@@ -182,61 +182,57 @@ double CalcWalletBalance(base::flat_map<std::string, double> wallets,
   return balance_minus_grant;
 }
 
-void RecordRewardsEnabledDuration(PrefService* prefs, bool rewards_enabled) {
-  base::Time enabled_timestamp = prefs->GetTime(prefs::kEnabledTimestamp);
-  auto enabled_duration = RewardsEnabledDuration::kNever;
+void RecordAdsEnabledDuration(PrefService* prefs, bool ads_enabled) {
+  base::Time enabled_timestamp = prefs->GetTime(prefs::kAdsEnabledTimestamp);
+  base::TimeDelta enabled_time_delta =
+      prefs->GetTimeDelta(prefs::kAdsEnabledTimeDelta);
+  AdsEnabledDuration enabled_duration;
 
   if (enabled_timestamp.is_null()) {
     // No previous timestamp, so record one of the non-duration states.
-    if (!rewards_enabled) {
-      // Rewards have been disabled with no previous timestamp.
-      // Probably they've been on since we started measuring.
-      // Ignore this interval since we can't measure it and treat it
-      // the same as never having enabled.
-      enabled_duration = RewardsEnabledDuration::kNever;
-    } else {
-      // Rewards have been enabled.
+    if (ads_enabled) {
+      // Ads have been enabled.
       // Remember when so we can measure the duration on later changes.
-      prefs->SetTime(prefs::kEnabledTimestamp, base::Time::Now());
-      enabled_duration = RewardsEnabledDuration::kStillEnabled;
+      prefs->SetTime(prefs::kAdsEnabledTimestamp, base::Time::Now());
+      enabled_duration = AdsEnabledDuration::kStillEnabled;
```

Comment:
Is this redundant with the `if (ads_enabled) { enabled_duration = AdsEnabledDuration::kStillEnabled; } else if ...` below?

Reply:
it is redundant. removed, thx